lib/rubocop/cop/sorbet/forbid_t_struct.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "rubocop"
+
+module RuboCop
+  module Cop
+    module Sorbet
+      # Disallows using `T::Struct`.
+      #
+      # `T::Struct` has runtime performance implications that can lead to around 5x slower code over the equivilant
```

Comment:
```suggestion
      # `T::Struct` has runtime performance implications that can lead to around 5x slower code over the equivalent
```

Reply:
Maybe better to remove the 5x part since that could change over time if Sorbet improves?